instructor/compiler.py

```diff
@@ -369,7 +373,7 @@ def experiment_temperatures(assignment):
             treatment = strain_protocol.treatment
             temperature = treatment.temperature
             ret[str(temperature.id)] = {
-                'name': str(temperature.degrees)
+                'name': str(temperature.degrees) + u'\u00b0C'
```

Comment:
Can you use the format string here and below as well for `start_time` and `duration`?